packages/samples/test/output/grpc-library-example/@typespec/openapi3/openapi.yaml

```diff
@@ -40,8 +40,10 @@ paths:
         Lists shelves. The order is unspecified but deterministic. Newly created
         shelves will not necessarily be added to the end of this list.
       parameters:
-        - $ref: '#/components/parameters/ListRequestBase.page_size'
-        - $ref: '#/components/parameters/ListRequestBase.page_token'
+        - &a1
```

Comment:
I wodner how OpenAPI3 paresrs will handle this.  The space savings is nice.  It does make individual operations that reuse these parameters a bit more difficult to consume by humans

Reply:
ah yeah oops, I explictly disabled yaml ref in the previous parser/serializer, I think we need the same here, will fix